modules/ROOT/pages/mtf-expecting-errors.adoc

```diff
@@ -0,0 +1,75 @@
+= Expected Errors
```

Comment:
Should this be Validating Expected Errors?

Reply:
Or is this a reference topic?
For multiple-task topics, use gerund in the H1.

Reply:
Thanks. It definitely helps to see these in context. I think you're right. No change!